packages/Webkul/Shop/src/Resources/views/products/view/gallery.blade.php

```diff
@@ -103,7 +103,7 @@ class="min-w-[450px] rounded-xl"
             </div>
 
             <!-- Product slider Image with shimmer -->
-            <div class="scrollbar-hide flex gap-8 overflow-auto 1180:hidden">
+            <div class="scrollbar-hide flex gap-8 overflow-auto 1180:hidden w-screen">
```

Comment:
Please reorder the class add 'w-screen' after 'flex'

Reply:
@shivendra-webkul Done